### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

With the objective of reducing notification noise, I'm moving myself to the PL section

### Does your PR introduce any breaking changes? If yes, please list them.

None

cc @carmocca @akihironitta @borda

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -54,28 +55,23 @@
...
-# Testing
-/tests/helpers/boring_model.py          @williamfalcon @tchaton @borda
+/src/pytorch_lightning/callbacks/callback.py @williamfalcon @awaelchli @ananthsub @carmocca
+/src/pytorch_lightning/core/datamodule.py    @williamFalcon @awaelchli @ananthsub @carmocca
+/src/pytorch_lightning/trainer/trainer.py    @williamfalcon @tchaton @awaelchli
+/src/pytorch_lightning/core/hooks.py         @williamfalcon @tchaton @awaelchli @ananthsub @carmocca
+/src/pytorch_lightning/core/module.py        @williamfalcon @tchaton @awaelchli
 
-/.github/CODEOWNERS              @williamfalcon
-/.github/approve_config.yml      @williamfalcon
```
Comment (carmocca) [inline .github/CODEOWNERS:77]: This file doesn't exist

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -54,28 +55,23 @@
...
+/tests/tests_app_examples                   @tchaton @awaelchli @manskx @hhsecond
 
 # Specifics
-/src/pytorch_lightning/trainer/connectors/logger_connector      @tchaton @carmocca
-/src/pytorch_lightning/trainer/progress.py                      @tchaton @awaelchli @carmocca
-
+/src/pytorch_lightning/trainer/connectors/logger_connector @tchaton @carmocca
+/src/pytorch_lightning/trainer/progress.py                 @tchaton @awaelchli @carmocca
 # API
-/src/pytorch_lightning/callbacks/base.py    @williamfalcon @awaelchli @ananthsub @carmocca
```
Comment (carmocca) [inline .github/CODEOWNERS:67]: This file was renamed

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -54,28 +55,23 @@
...
-/src/pytorch_lightning/core/datamodule.py   @williamFalcon @awaelchli @ananthsub @carmocca
-/src/pytorch_lightning/trainer/trainer.py   @williamfalcon @tchaton @awaelchli
-/src/pytorch_lightning/core/hooks.py        @williamfalcon @tchaton @awaelchli @ananthsub @carmocca
-/src/pytorch_lightning/core/lightning.py    @williamfalcon @tchaton @awaelchli
```
Comment (carmocca) [inline .github/CODEOWNERS:71]: This file was renamed

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -54,28 +55,23 @@
...
-
-# Testing
-/tests/helpers/boring_model.py          @williamfalcon @tchaton @borda
```
Comment (carmocca) [inline .github/CODEOWNERS:74]: This file doesn't exist

**Review by Borda — APPROVED** [review]

File: src/pytorch_lightning/__about__.py [hunk 5]
```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 import time
 
-# __version__ = "1.7.0"
```
Comment (akihironitta) [inline src/pytorch_lightning/__about__.py:16]: Is the change here on purpose?

Comment (Borda) [reply src/pytorch_lightning/__about__.py:16]: we can remove this line

**Pull request merged**